core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -355,6 +355,8 @@ private static Map<SqlKind, Policy> createPolicyMap() {
     map.put(SqlKind.CHECKED_MINUS_PREFIX, Policy.ANY);
     map.put(SqlKind.CHECKED_TIMES, Policy.ANY);
     map.put(SqlKind.CHECKED_DIVIDE, Policy.ANY);
+    map.put(SqlKind.DIVIDE_0_NULL, Policy.AS_IS);
+    map.put(SqlKind.MOD_0_NULL, Policy.AS_IS);
```

Comment:
We could override the `SqlOperator.getStrongPolicyInference` for the new operators and avoid the need for introducing a new SqlKind entry.